src/Accord.Services/Reminder/DeleteReminder.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Accord.Domain;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Accord.Services.Reminder
+{
+    public sealed record DeleteReminderRequest(ulong DiscordUserId, int ReminderId) : IRequest<ServiceResponse>;
+
+    public sealed record DeleteAllRemindersRequest(ulong DiscordUserId) : IRequest<ServiceResponse>;
+
+    public class DeleteReminderHandler :
```

Comment:
I like the reuse here

Reply:
i'm guessing that's good, i can almost never understand :P